include/aws/io/socket.h

```diff
@@ -78,7 +98,7 @@ typedef void(aws_socket_on_connection_result_fn)(struct aws_socket *socket, int
  * A user may want to call aws_socket_set_options() on the new socket if different options are desired.
  *
  * new_socket is not yet assigned to an event-loop. The user should call aws_socket_assign_to_event_loop() before
- * performing IO operations.
+ * performing IO operations. The user must call `aws_socket_release()` when they're done with the socket, to free it.
```

Comment:
There is no `aws_socket_release()`? Is this something that existed in some old commit and deleted since?